src/pages/extending-chat-widget/customer-sdk/index.mdx

```diff
@@ -89,16 +89,16 @@ You will need to provide your licenseId and clientId when creating a Customer SD
-| -------------------- | ------------------ | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| licenseId            | number             |         | Your license number, you receive this value when creating a new livechat account.                                                                                                                                                                                   |
-| clientId             | string             |         | Your client id, you receive this value when you register your application in the LiveChat Developer Console.                                                                                                                                                        |
-| autoConnect          | boolean            | true    | Optional; should the library try to reconnect on it's own.                                                                                                                                                                                                          |
-| groupId              | number             | 0       | Optional; the id of the group you wish to connect to.                                                                                                                                                                                                               |
-| region               | 'dal' \| 'fra'     | 'dal'   | Optional; the server region your license is at.                                                                                                                                                                                                                     |
-| redirectUri          | string             |         | Optional; should only be used inside ReactNative, this is the URI which your webview is redirected to after authorization.                                                                                                                                          |
-| customerDataProvider | () => CustomerData |         | Optional; should only be used if you need to send customer data during login. In general, [`updateCustomer()`](#updateCustomer) should be prefered for sending customer data.                                                                                       |
-| identityProvider     | () => CustomerAuth |         | Optional; allows for providing own instance of the CustomerAuth object which contains the customer access token handlers. See [Custom Identity Provider](https://developers.livechat.com/docs/extending-chat-widget/custom-identity-provider) for more information. |
```

Comment:
Why do we remove `identityProvider` from the table here? Is it by accident? 🤔

Reply:
Yes, copy-paste went wrong. 🙈 I wanted to avoid a lot of linting updates during that change, hence I had to work around it a bit. 😅